example/akamai.jsonnet

```diff
@@ -47,6 +47,24 @@ test.suite {
       ],
     },
 
+    // This test shows how to add a delay before the execution of a test
+    // This can be useful, if previous tests or actions require a delay before the next the can be executed
+    test.case {
+      name: 'Delay of test execution by 3 seconds',
+      pre:: [test.utils.delay(3000)], // Add a delay of 3s before the test is executed
+      request: test.GET({
```

Comment:
sorry to bother you with this, but can you please replace this syntax with:

`      request: test.POST('https://{{httpbin}}/response-headers?x-cache=TCP_MISS')`

The full request syntax is useful but not necessary here, and I would prefer folks copy pasted simpler examples.

Reply:
I just created a new commit. Please check again.